### ISSUE TITLE
Home button does nothing in-game

### ISSUE THREAD

**Issue by Licho1** [ZeroK-RTS/Chobby#394]
Join a game,
rejoin,
Press f11,
Click "home" -> nothing happens.

**Comment by GoogleFrog** [comment]
Cannot reproduce. When I click "Home" the zero-k.info page open on the browser on my other monitor (the one I am typing on now). There is a known issue that the OS window focus does not change when chobby tells the wrapper to open a URL. I expect that if you were ingame you did not notice the new browser window opening behind it. I don't know if/where it is reported though.

**Comment by Licho1** [comment]
yes i had no idea .. i guess we can switch the focus then .. that is wrapper issue though.
